send/ad_mu

```diff
@@ -95,16 +93,6 @@ close $file_g;
 my $namespace = $ARGV[1];
 chomp($namespace);
 
-# create service lock
-my $lock = ScriptLock->new($facility_name . "_" . $service_name . "_" . $namespace);
-($lock->lock() == 1) or die "Unable to get lock, service propagation is already running or previous run failed.";
```

Comment:
I think you should keep our standard lock `ScriptLock` as it prevents running the script concurrently and has no effect on failed status of the script (error lock which can be removed). See eg. `ad_group_vsup` service script.

Reply:
Fixed